bin/kafka-server-stop.sh

```diff
@@ -21,7 +21,7 @@ if [[ $(uname -s) == "OS/390" ]]; then
     fi
     PIDS=$(ps -A -o pid,jobname,comm | grep -i $JOBNAME | grep java | grep -v grep | awk '{print $1}')
 else
-    PIDS=$(ps ax | grep -i 'kafka\.Kafka' | grep java | grep -v grep | awk '{print $1}')
+    PIDS=$(ps ax | grep 'kafka\.Kafka' | grep java | grep -v grep | awk '{print $1}')
```

Comment:
Is there any reason we can't make it more specific by requiring an exact match with surrounding spaces: `' kafka\.Kafka '`?

Reply:
@lbradstreet , yes, that'd be better! Thanks for suggestion. I've updated in this commit: https://github.com/apache/kafka/pull/9179/commits/99bf042dc346cca5061e7f0bbcb30b1d736162f6. Thanks.